Makefile

```
@@ -56,17 +56,17 @@ deploy: az-login ## Deploy all infrastructure
 deploy-core: az-login ## Deploy core infrastructure
 	$(call target_title, "Deploy Core Infrastructure") \
 	&& . ${MAKEFILE_DIR}/scripts/load_env.sh \
-	&& terragrunt apply ${MAKEFILE_DIR}/infrastructure/core
+	&& terragrunt run-all apply --terragrunt-working-dir ${MAKEFILE_DIR}/infrastructure/core --terragrunt-non-interactive
```

```suggestion
		&& cd ${MAKEFILE_DIR}/infrastructure/serve \
	&& terragrunt run-all apply --terragrunt-include-external-dependencies --terragrunt-non-interactive
```
and likewise for the others works ok

---

Works like a charm, thanks!